content/concepts/transports/webrtc.md

```diff
@@ -1,9 +1,150 @@
+
+1. The browser sends a `ClientHello` message to advertise its supported cipher suites
+   and other protocol options to the server.
+2. The server responds with a `ServerHello` message to indicate which cipher suite
+   and other options it has selected for communication.
+3. The server sends its certificate along with a `ServerKeyExchange` message to the client
+   that includes the necessary information for the client to generate a premaster secret.
+4. The browser verifies the server's certificate and sends a `ClientKeyExchange` message,
+   which contains the premaster secret encrypted using the server's public key.
+5. The server decrypts the premaster secret and uses it, along with its private key, to
+   generate the session keys.
+
+A successful DTLS handshake only provides confidentiality and integrity. Authenticity is
+achieved by succeeding a [Noise handshake](../secure-comm/noise) following
+the DTLS handshake. Messages on each `RTCDataChannel` are framed using a message-framing
+mechanism described
+[here](https://github.com/libp2p/specs/blob/master/webrtc/README.md#multiplexing).
+
+<!-- TO ADD: DIAGRAMS ONCE READY + CONTEXT -->
+
+### Coming soon: Browser-to-Browser
+
+Eventually, libp2p will have support for two-way communication between two
+browsers in real-time.
+
+The technical specification and initial implementations of WebRTC
+Browser-to-Browser connectivity is planned for release in early 2023.
+Track the progress [here](https://github.com/libp2p/specs/issues/475).
+
+{{< alert icon="💡" context="note" text="See the WebRTC <a class=\"text-muted\" href=\"https://github.com/libp2p/specs/blob/master/webrtc/README.md\">technical specification</a> for more details." />}}
+
+## Comparing WebRTC and WebTransport
+
+While WebRTC and WebTransport are both web-based approaches that enable real-time
+communication between nodes, there are key differences. WebRTC supports peer-to-peer
+connections, while WebTransport only supports client-server connections.
+
+The underlying protocols in WebRTC and WebTransport are different, although both
+protocols share many of the same properties. WebTransport is also an
+alternative to the data channels available in WebRTC.
+
+WebRTC is also more complex, as there are many underlying protocols involved in order
+to create an active transport, as opposed to WebTransport, that uses QUIC.
+
+When connecting to a WebSocket server or when using plain TCP or QUIC connections,
+browsers require the server to present a TLS certificate signed by a trusted CA
+(certificate authority). Few nodes have such a certificate. One method to overcome
+this is to use the [WebTransport](webtransport) browser API that offers a way to
+accept a server's certificate by checking the (SHA-256) hash of the certificate.
+
+However, a certificate is still needed, even if it is "just" self-signed.
+The browser must also know the certificate hash. WebRTC can overcome this and
+does not require a trusted certificate.
```

Comment:
I think this is misleading. WebRTC still requires a TLS certificate. Both WebTransport and WebRTC can operate on top of self-signed certificates. WebRTC needs to know the certificate hash of the remote as well.